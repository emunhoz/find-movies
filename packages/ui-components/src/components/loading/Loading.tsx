import React from 'react'
import * as S from './Loading.style'

export const Loading = () => <S.Loading><div className="spinner" data-testid="loading"></div></S.Loading>
