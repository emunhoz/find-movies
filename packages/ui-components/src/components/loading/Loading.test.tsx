import React from 'react'
import { render } from '../../utils/test-utils'
import { Loading } from './Loading'

describe('[Component]: Loading', () => {
  test('render the component', () => {
    const { asFragment } = render(<Loading />)
    expect(asFragment()).toMatchSnapshot()
  })
})
