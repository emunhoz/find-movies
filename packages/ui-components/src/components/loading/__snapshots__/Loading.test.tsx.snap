// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`[Component]: Loading render the component 1`] = `
<DocumentFragment>
  <div
    class="sc-bdfBwQ hBrFHp"
  >
    <div
      class="spinner"
      data-testid="loading"
    />
  </div>
</DocumentFragment>
`;
